projects/vdk-plugins/vdk-jobs-troubleshooting/src/vdk/plugin/jobs_troubleshoot/troubleshoot_configuration.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from vdk.internal.core.config import ConfigurationBuilder
+
+TROUBLESHOOT_UTILITIES_TO_USE = "TROUBLESHOOT_UTILITIES_TO_USE"
+TROUBLESHOOT_PORT_TO_USE = "TROUBLESHOOT_PORT_TO_USE"
+
+
+def add_definitions(config_builder: ConfigurationBuilder):
+    """
+    Here we define what configuration settings are needed for the
+    Jobs Troubleshooting plugin with reasonable defaults.
+    """
+    config_builder.add(
+        key=TROUBLESHOOT_UTILITIES_TO_USE,
+        default_value=None,
+        description="""
+            An unordered comma-separated list of strings, indicating what
+            troubleshooting utilities are to be used. E.g., "utility1,utility2".
+            """,
+    )
+    config_builder.add(
+        key=TROUBLESHOOT_PORT_TO_USE,
+        default_value=8080,
```

Comment:
Let's not use 8080. It's such a common port that it's easy to conflict. 8783  or something similarly random.

Reply:
Changed, thanks